src/writeData/subscriptions/components/BrokerFormContent.tsx

```diff
@@ -185,6 +189,7 @@ const BrokerFormContent: FC<Props> = ({
                 )}
               />
             </div>
+            {/* show this iff brokerHost is populated */}
```

Comment:
do we need this comment?

Reply:
nope.. were just a note. thanks for catching it